pkg/config/node/minimum_kubelet_version.go

```diff
@@ -0,0 +1,80 @@
+package node
+
```

Comment:
It's not a big deal, but I think `pkg/config` is meant for client configuration. How about moving this code to `pkg/apiserver`?

Reply:
done